ibmcloud/README.md

```diff
@@ -91,29 +188,24 @@ As usual, you need to create `terraform.tfvars` to specify parameter values. The
 ibmcloud_api_key = "<your API key>"
 ssh_key_name = "<your SSH key name>"
 cluster_name = "<cluster name>"
+primary_subnet_name = "<name of your primary subnet>" OR primary_subnet_id = "<ID of your primary subnet>"
+primary_security_group_name = "<name of your primary security group>" OR primary_security_group_id = "<ID of your primary security group>"
+vpc_name = "<name of your VPC>" OR vpc_id = "<ID of your VPC>"
 ```
+
 If you don't have your public key already configured in IBM Cloud you can add
 ```
 ssh_pub_key = "<your SSH public key>"
 ```
-> **Hint:** In order to create the cluster based on a different type of VSI image you can overwrite more parameters here e.g. to create a **s390x** based cluster add follow two lines to the `terraform.tfvars` file
```

Comment:
Optional/idea: I wonder if it is still worth repeating this section in the Cluster creation, or point to the section where it is mentioned in the e2e process to highlight that it's relevant at this stage?

Reply:
I've added the hint back.